<ui version="4.0" >
 <class>MinerMainWindow</class>
 <widget class="QMainWindow" name="MinerMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Mining results viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>29</y>
     <width>800</width>
     <height>549</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>9</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Forms</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Scoring:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="scoringComboBox" >
            <item>
             <property name="text" >
              <string>Suspicion</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Suspicion * f(suspObs)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Suspicion * f(uniqSuspSents)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Suspicion * ln(f(suspObs))</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Suspicion * ln(f(uniqSuspSents))</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Regex:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="regExpLineEdit" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="formsTreeWidget" >
          <property name="selectionMode" >
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <column>
           <property name="text" >
            <string>Form</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>Score</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="removeFormPushButton" >
            <property name="text" >
             <string>Remove form</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>Form information</string>
          </property>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>9</number>
           </property>
           <item>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="2" column="0" >
              <widget class="QLabel" name="label_4" >
               <property name="text" >
                <string>Unparsable frequency:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLabel" name="suspicionLabel" >
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLabel" name="freqLabel" >
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1" >
              <widget class="QLabel" name="suspFreqLabel" >
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Frequency:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Suspicion:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" >
              <widget class="QLabel" name="label_6" >
               <property name="text" >
                <string>Unique sentences:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" >
              <widget class="QLabel" name="uniqSentsFreqLabel" >
               <property name="text" >
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4" >
          <property name="title" >
           <string>Selected sentence:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="QLineEdit" name="sentenceLineEdit" />
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>Unparsable sentences</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QCheckBox" name="allSentenceMatchCheckBox" >
             <property name="text" >
              <string>Match all unparsable sentences</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout" >
             <item row="1" column="0" >
              <widget class="QLabel" name="label_7" >
               <property name="text" >
                <string>Regex:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QLineEdit" name="sentenceRegExpLineEdit" />
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="sentenceListWidget" />
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="preferencesAction" />
    <addaction name="separator" />
    <addaction name="quitAction" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="copyAction" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>578</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="quitAction" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="newAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>New...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Open...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="copyAction" >
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="preferencesAction" >
   <property name="text" >
    <string>Pref&amp;erences...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>activated()</signal>
   <receiver>MinerMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
